frontend/components/notifications/management/NotificationsManagementDashboards.vue

```diff
@@ -353,14 +354,17 @@ function getTypeIcon(type: DashboardType) {
           header-class="action-col"
         >
           <template #body="slotProps">
-            <!-- TODO: once we have our api check how to identify 'deleted' rows -->
             <div class="action-row">
-              <FontAwesomeIcon
-                :disabled="!slotProps.data.subscriptions?.length ? true : null"
-                :icon="faTrash"
-                class="link"
+              <BcButtonIcon
+                :screenreader-text="`Delete all notifications for ${slotProps.data.dashboard_name}`"
                 @click="onEdit('delete', slotProps.data)"
-              />
+              >
+                <FontAwesomeIcon
+                  :disabled="!slotProps.data.subscriptions?.length ? true : null"
```

Comment:
plz have a look what the `disabled` attribute wanted to achieve before 😮

Reply:
asked in the `api channel` if they have thoughts on how to `convey` if there are `notifications to be deleted` in a `convenient way`